src/pages/HomePage/HomePage.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Heading = styled.h1`
```

Comment:
I like to keep my styled components in a separate file, for example Homepage.style.js

Imo it's a bit cleaner and easier to look at when the components grow. But it's a matter of taste and you can change that later on as well. 

Feel free to resolve this conversation